### PR TITLE
Working on order products status

### PR DESCRIPTION
**Pull request by aadi163**

### CODE REVIEW

File: app/views/homepage/index.html.erb [hunk 1]
```diff
@@ -7,5 +7,8 @@
   </div>
   <%= render "carousal" %>
   <div><%= render "productcart" , products: @products%></div>
+  <%= render "footer_carousal" %>
+  
+
```
Comment (Kapil0187) [inline app/views/homepage/index.html.erb]: remove extra lines from here

File: app/views/homepage/filter_products.html.erb [hunk 2]
```diff
@@ -7,5 +7,7 @@
   </div>
   <%= render "carousal" %>
   <%= render "productcart" , products: @categoryProducts%>
+  <%= render "footer_carousal" %>
+
```
Comment (Kapil0187) [inline app/views/homepage/filter_products.html.erb]: remove extra lines from here

File: app/views/orders/index.html.erb [hunk 3]
```diff
@@ -25,12 +25,9 @@
...
                     turbo_confirm: "Are you sure?"
             },class:"btn btn-danger" %></td>
-      <td><% if order.product.product_status.present? %>
-            <%=order.product.product_status.status%>
-          <% else %>
-            Pending...
-          <% end %>
-      </td>
+      <% if order.product_status.present? %>
+      <td><h6><%=order.product_status.status%></h6></td>
```
Comment (ShahirMansuri786) [inline app/views/orders/index.html.erb]: please correct indent use two spaces

File: app/views/seller_homepage/seller_orders.html.erb [hunk 4]
```diff
@@ -30,14 +30,11 @@
         <span> Contact No. <%=item.useraddress.phonenumber%> </span>
       </td>
       <td>
-        <% if item.product.product_status.present? %>
-          <%=item.product.product_status.status%>
-        <% else %>
-          Pending...  
+        <% if item.product_status.present? %>
+        Current Status <h5> <%=item.product_status.status%> </h5>
```
Comment (ShahirMansuri786) [inline app/views/seller_homepage/seller_orders.html.erb]: if you are inside the block so use proper indent. here also incorrect indent

**Review by Kapil0187 — CHANGES_REQUESTED** [review]

File: app/views/homepage/index.html.erb (hunk 1 above)
Comment (aadi163) [reply app/views/homepage/index.html.erb]: ok sure I will remove

File: app/views/seller_homepage/seller_orders.html.erb (hunk 4 above)
Comment (aadi163) [reply app/views/seller_homepage/seller_orders.html.erb]: ok I will correct this

**Pull request merged**